debian/rules

```diff
@@ -55,9 +54,18 @@ CONFIG_ARM64 := ${CONFIG_ARM_COMMON} \
 	--cross-prefix=/usr/bin/aarch64-linux-gnu- \
 
 CONFIG_x86 := --arch=amd64 \
+	--enable-libzimg \
+	--enable-opencl \
+	--enable-vaapi \
 	--enable-amf \
+	--enable-libmfx \
+	--enable-vdpau \
+	--enable-cuda \
+	--enable-cuda-llvm \
+	--enable-cuvid \
 	--enable-nvenc \
 	--enable-nvdec \
+	--enable-ffnvcodec \
 	--enable-vaapi \
```

Comment:
```suggestion
```

It's redundant.

Reply:
Whoops missed that one, but that's fine, a bit of redundancy doesn't hurt.